config/keybindings.go

```diff
@@ -1,24 +1,216 @@
 package config
 
 import (
-	"errors"
+	"strings"
 
+	"github.com/gdamore/tcell"
 	"github.com/spf13/viper"
 )
 
-// KeyToNum returns the number on the user's keyboard they pressed,
-// using the rune returned when when they press Shift+Num.
-// The error is not nil if the provided key is invalid.
-func KeyToNum(key rune) (int, error) {
-	runes := []rune(viper.GetString("keybindings.shift_numbers"))
-	for i := range runes {
-		if key == runes[i] {
-			if i == len(runes)-1 {
-				// Last key is 0, not 10
-				return 0, nil
+const (
+	CmdInvalid = 0
```

Comment:
I think the code would be more clear if you defined something like `type Command int`, and made all these consts of that type. Then later lines like `func GetKeyBinding(cmd int) string` would become `func GetKeyBinding(cmd Command) string` which I find more readable and foolproof.